frontend/src/components/Table/Table.scss

```diff
@@ -55,7 +55,7 @@
   }
   .td,
   .th {
-    height: 5rem;
```

Comment:
statically setting the height breaks the table in case of vertical overflow. setting min-height instead ensures that the table heights are consistent in the case of normal content, but also allows for vertical overflow.

Reply:
+1